.github/workflows/cancel-previous-pr-workflows.yml

```diff
@@ -0,0 +1,11 @@
+name: Cancel Previous PR Workflows
+on:
+  pull_request:
+    branches: [ main ]
+jobs:
+  cancel:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: styfle/cancel-workflow-action@0.8.0
+      with:
+        workflow_id: "auto-update-dependencies.yml, build-agent-container.yml, build-anomaly-detection-app-container.yml, build-controller-container.yml, build-onvif-video-broker-container.yml, build-opcua-monitoring-broker-container.yml, build-opencv-base-container.yml, build-rust-code.yml, build-rust-crossbuild-container.yml, build-udev-video-broker-container.yml, build-video-streaming-app-container.yml, build-webhook-configuration-container.yml, cancel-previous-pr-workflows.yml, check-rust.yml, check-versioning.yml, run-helm.yml, run-tarpaulin.yml, run-test-cases.yml, security-audit.yml"
```

Comment:
"cancel-previous-pr-workflows.yml" is included here too. Is the intention to cancel existing "cancel workflow" and then the new one will take care of cancelling other older ones? (too many "cancel" in my statement 😄)

Reply:
I just put all workflows here. i dont really have a strong preference around including or excluding `cancel-previous-pr-workflows.yml`. If you are updating the workflow in a PR then you could cancel previous instances of it, but it is such a short workflow that its not consuming time.